packages/bpk-docs/src/components/DocsPageBuilder/NeoDocsPageBuilder.js

```diff
@@ -195,15 +195,15 @@ const NeoDocsPageBuilder = props => {
 
   return (
     <BpkContentContainer
-      className={getClassName(
-        `bpkdocs-content-page--${
-          sections.length % 2 === 0 ? 'even' : 'odd'
-        }-sections`,
-      )}
+      className={
+        props.wrapped
+          ? getClassName(`bpkdocs-content-page--wrapped`)
```

Comment:
Personally I think it's a bit weird that we set the background of the whole page gray and then override the white bits. I think the code would be a bit less hacky if the background was white to begin with. Then it would only the the `AlternatingPageContent` that would have to "override" the white background

Reply:
I see what you mean, but the background colour of the entire site is set to grey, because the home page is grey. So to change this would mean changing all that too, which I'm not sure we want to do.